src/mdx-pages/guides/webpack.mdx

```diff
@@ -3,52 +3,12 @@ title: Webpack and Video.js
 category: integrations
 ---
 
-Video.js, and the playback technologies such as videojs-contrib-hls all work in a Webpack based build environment. Here are several configuration changes specific to Webpack that will get you up and running.
+Video.js and official libraries and plugins all work in a Webpack-based build environment. These are several configuration changes specific to Webpack that will get you up and running.
```

Comment:
I think that last line might be a bit confusing since we only describe the one CSS requirement. Maybe we can specify something like: "You'll only need to configure the style-loader in Webpack to get up and running."?